db/migrate/20160905034718_change_project_steps_to_timeline_entries.rb

```diff
@@ -0,0 +1,11 @@
+class ChangeProjectStepsToTimelineEntries < ActiveRecord::Migration
+  def change
+    rename_table  :project_steps, :timeline_entries
+    rename_column :project_logs,  :project_step_id, :timeline_entry_id
```

Comment:
You know I realized I was wrong about this. Since logs can only belong to steps, not all timeline entries, I think this should stay the way it was. Sorry! Unless you think there is a reason... Can you change it back?